xml/System.Data/DataRowView.xml

```diff
@@ -539,7 +539,17 @@
         <summary>Gets a value indicating whether the current <see cref="T:System.Data.DataRowView" /> is identical to the specified object.</summary>
         <returns>
           <see langword="true" /> if <paramref name="object" /> is a <see cref="T:System.Data.DataRowView" /> and it returns the same row as the current <see cref="T:System.Data.DataRowView" />; otherwise <see langword="false" />.</returns>
-        <remarks>To be added.</remarks>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+Checks for same reference instead of equivalent <xref:System.Data.DataRowView.DataView> or <xref:System.Data.DataRowView.Row>.
+ 
+Necessary for `ListChanged` event handlers to use data structures that use the default to <xref:System.Object.Equals(System.Object)> instead of <xref:System.Object.ReferenceEquals(System.Object,System.Object)> to understand if they need to add a <xref:E:System.Data.DataRowView.PropertyChanged> event handler.
```

Comment:
This sentence (except it lacks a noun) makes no sense.

Reply:
Also sounds like an internal remark. I'll remove it.